undertow/src/main/resources/schema/wildfly-undertow_2_0.xsd

```diff
@@ -450,7 +450,7 @@
     <xs:complexType name="modClusterType">
         <xs:attribute name="name" use="required" type="xs:string"/>
         <xs:attribute name="management-socket-binding" type="xs:string" use="required"/>
-        <xs:attribute name="advertise-socket-binding" type="xs:string" use="required"/>
+        <xs:attribute name="advertise-socket-binding" type="xs:string" use="optional"/>
```

Comment:
My comments was about old XSDs. The change to the new XSD to match the model is fine.